src/routes/scroll-to-top/scroll-to-top.test.tsx

```diff
@@ -13,6 +16,10 @@ const defaultProps = {
 };
 
 describe('ScrollToTop', () => {
+  afterEach(() => {
+    vi.clearAllMocks();
+    vi.resetAllMocks();
+  });
```

Comment:
Ideally this would be done on every single test on every single file. Is there a way to do it?